### PR TITLE
DW-3936 - Creating a QA job for this pipeline, tidying up Account number refs

### PR DESCRIPTION
**Pull request by benjaminsherwood**
Signed-off-by: Benjamin Sherwood <benjaminsherwood@digital.uc.dwp.gov.uk>

### CODE REVIEW

File: ci/jobs/analytical-dataset-generation-pr.yml [hunk 1]
```diff
@@ -30,6 +30,7 @@ jobs:
             TF_CLI_ARGS_apply: -lock-timeout=300s
             TF_CLI_ARGS_plan: -lock-timeout=300s
             TF_VAR_slack_webhook_url: ((slack.webhook_url))
```
Comment (dan-hill2802) [inline ci/jobs/analytical-dataset-generation-pr.yml:32]: missed one 😛

File: ci/jobs/analytical-dataset-generation-dev.yml [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+    - get: emr-launcher-release
+      trigger: true
+  - .: (( inject meta.plan.terraform-bootstrap ))
+  - .: (( inject meta.plan.create-aws-profiles ))
+    config:
+      params:
+        AWS_ACC: ((aws_account.development))
+  - .: (( inject meta.plan.terraform-apply ))
+    params:
+      TF_WORKSPACE: dev
```
Comment (dan-hill2802) [inline ci/jobs/analytical-dataset-generation-dev.yml:16]:
```suggestion
      TF_WORKSPACE: default
```

File: ci/jobs/analytical-dataset-generation-dev.yml [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
+  - .: (( inject meta.plan.terraform-plan ))
+    params:
+      TF_WORKSPACE: dev
```
Comment (dan-hill2802) [inline ci/jobs/analytical-dataset-generation-dev.yml:19]:
```suggestion
      TF_WORKSPACE: default
```

File: ci/jobs/synthetic-data-ingestion-dev.yml [hunk 4]
```diff
@@ -15,17 +15,17 @@ jobs:
...
     config:
       params:
-        AWS_ACC: "{{accounts['development']}}"
-        SYNTHETIC_RAWDATA_AWS_ACC: "{{accounts['management-dev']}}"
+        AWS_ACC: ((aws_account.development))
+        SYNTHETIC_RAWDATA_AWS_ACC: ((aws_account.management-dev))
         SYNTHETIC_RAWDATA_PREFIX: "synthetic-data/raw"
         SYNTHETIC_ENCDATA_PREFIX: "synthetic-data/encrypted"
-        IS_SYNTHETIC_DATA_INGESTION: "TRUE"
+        IS_SYNTHETIC_DATA_INGESTION: "TRUE"
```
Comment (dan-hill2802) [inline ci/jobs/synthetic-data-ingestion-dev.yml:31]: you've accidentally removed the new line

**Comment by dan-hill2802** [comment]
I appreciate the code clean up and tech debt addressed.

File: ci/jobs/analytical-dataset-generation-dev.yml (hunk 2 above)
Comment (mattburgess) [reply ci/jobs/analytical-dataset-generation-dev.yml:16]: Can we just remove `TF_WORKSPACE` from the dev pipeline. I don't think we set it in other dev pipelines, do we?

**Pull request merged**